VimeoUpload/Descriptor System/DescriptorManager.swift

```diff
@@ -317,18 +317,11 @@ open class DescriptorManager: NSObject
     
     // MARK: Public API
     
-    open func handleEventsForBackgroundURLSession(identifier: String, completionHandler: @escaping VoidClosure) -> Bool
+    open func handleEventsForBackgroundURLSession(identifier: String, completionHandler: @escaping VoidClosure)
```

Comment:
Do we use the passed in identifier anymore? If not, do we need to check that we're handling the right session in the example projects (should we still return here, and if so, should this be called `ifNeeded`?

Reply:
@nicolelehrer yes we should remove the `identifier` argument. To help third-party users to check if the descriptor manager should handle background session's events, I've added a new method in `DescriptorManager` for that purpose. I think the library should not absorb the check but instead should let the client decide.